restore/restore.go

```diff
@@ -140,27 +142,155 @@ func (c *RestoreCommand) StartRestore(cliConnection plugin.CliConnection, servic
 	defer resp.Body.Close()
 	body, err := ioutil.ReadAll(resp.Body)
 
-	if resp.Status != "202 Accepted" {
-		fmt.Println(AddColor("FAILED", red))
-		var message string = string(body)
-		var parts []string = strings.Split(message, ":")
-		fmt.Println("Error - ",parts[3])
+	var respObject map[string]interface{}
+
+	if err := json.Unmarshal(body, &respObject); err != nil {
+		fmt.Println(err)
+	}
+
+	if respStatus, flag := respObject["status"].(float64); flag != false {
```

Comment:
That is the type of the variable/property returned in the response. If I cast it as an integer, it will throw an error saying the property type is `float64`

Reply:
okay.